.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,21 +1,21 @@
 ---
-name: Feature request
-about: Suggest a new idea for Jamulus (not an improvement of an existing function)
+name: Feature
+about: Agreed specification for a feature or enhancement
 title: ''
-labels: 'feature request'
+labels: 'feature'
 assignees: ''
 ---
 
-<!-- Thanks for opening a new feature request! Please keep in mind that Jamulus achieves low latency audio by reducing complexity and following the "Keep it simple and stupid" approach. Adding new features may result in more processing and maintenance overhead, so please be sure that your feature can not be achieved by different approaches/software too. Make it clear, why here. -->
+<!-- Thanks for opening a new feature request! -->
```

Comment:
Should this skip "request" as well if we intentionally remove it from the label as well?
```suggestion
<!-- Thanks for suggesting a new feature! -->
```

Or
```suggestion
<!-- Thanks for specifying a new feature! -->
```
?

Reply:
Yes I was wondering about that too. I guess it's logical :-)